notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<p>These install would be required for Workbench as well. I would remove the if not IS_...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I tested it like this and didn't need them for user managed notebooks.</p>

Reply:
<p>I'm happy to add them if you'd prefer for minimizing cross-notebook diffs but I didn't seem to need them in my testing.</p>

Reply:
<p>Let's take out the IF statement. </p>

Reply:
done